src/containers/reports/ReportsReducer.js

```diff
@@ -36,6 +32,10 @@ export default function reportReducer(state :Map<*, *> = INITIAL_STATE, action :
 
   switch (action.type) {
 
+    case CLEAR_CRISIS_TEMPLATE: {
+      return state;
```

Comment:
is this supposed to return INITIAL_STATE ?

Reply:
Indeed. Thanks for catching that.